data/fields/marker.json

```
@@ -1,5 +1,33 @@
 {
     "key": "marker",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+            "aerial": {
+                "title": "Aerial",
+                "description": "Designed to be seen by aerial inspection."
+            },
+            "ground": {
+                "title": "Ground",
+                "description": "Flat marker fixed directly onto the ground."
```

“Fixed” is a potentially confusing word, and it’s assumed for a marker anyways. Maybe something like this?

```suggestion
                "description": "Flat marker laid in the ground."
```

Or maybe:

```suggestion
                "description": "Laid flat on the ground in a fixed location."
```